src/CanvasPaintings/Mapper/Identity/Identifiable.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Mapper.Identity
+{
+    public class Identifiable
+    {
+        private static readonly char[] Numbers = "23456789".ToCharArray();                   // not 1, 0
+        private static readonly char[] Letters = "abcdefghjkmnpqrstuvwxyz".ToCharArray();    // not i, l, o
+        private static readonly char[] All = [.. Numbers, .. Letters];
+
+        public static string Generate(int length = 8, bool letterFirst = true)
+        {
+            Random random = new Random();
+            var sb = new StringBuilder(length);
+            for (int i = 0; i < length; i++)
+            {
+                if (letterFirst && i == 0)
+                {
+                    sb.Append(Letters[random.Next(Letters.Length)]);
+                }
+                else
+                {
+                    sb.Append(All[random.Next(All.Length)]);
+                }
+            }
+            return sb.ToString();
```

Comment:
I wonder if we could hash the canvas ID to get an identifier? (Might simplify PUT/update requests)